pom.xml

```diff
@@ -55,7 +66,11 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.xmlunit</groupId>
+            <artifactId>xmlunit-core</artifactId>
```

Comment:
why do you need this dependency?

Reply:
I added this dependence because I had a warning for "spring-boot-starter-test": 

"Vulnerability found in dependency:
GHSA-chfm-68vv-pvw5 2.0 Transitive XMLUnit for Java has Insecure Defaults when Processing XSLT Stylesheets
Show details…"

details: 
"Dependency maven:org.xmlunit:xmlunit-core:2.9.1 is vulnerable
Upgrade to 2.10.0"